samples/functions/js/iot-hub-data-visualization/index/index.js

```diff
@@ -1,29 +1,27 @@
 var fs = require("fs");
 var path = require("path");
 
-module.exports = function (context) {
+module.exports = function (context, request) {
   var index = path.join(
     context.executionContext.functionDirectory,
     "public",
-    "index.html"
+    request.query.merge
```

Comment:
Can we just use different file names instead of query strings?

Reply:
Or maybe we only keep one?

Reply:
One is also fine.